packages/nodes-base/nodes/Gitlab/Gitlab.node.ts

```diff
@@ -1707,6 +1759,19 @@ export class Gitlab implements INodeType {
 						}
 						endpoint = `${baseEndpoint}/repository/tree`;
 					}
+				} else if (resource === 'mergeRequest') {
```

Comment:
Could you add some tests for these new operations?

Unfortunately we don't have tests yet for his node yet. You can check out the `Youtube.node.test.ts` or `Github.node.test.ts` for examples on how we test similar nodes.